crates/stackable-webhook/src/tls.rs

```diff
@@ -171,32 +172,98 @@ impl TlsServer {
             // trait function on `IntoMakeServiceWithConnectInfo`
             let tower_service = router.call(remote_addr).await.unwrap();
 
-            tokio::spawn(async move {
-                // Wait for tls handshake to happen
-                let Ok(tls_stream) = tls_acceptor.accept(tcp_stream).await else {
-                    trace!(%remote_addr, "error during tls handshake connection");
-                    return;
-                };
-
-                // Hyper has its own `AsyncRead` and `AsyncWrite` traits and doesn't use tokio.
-                // `TokioIo` converts between them.
-                let tls_stream = TokioIo::new(tls_stream);
-
-                // Hyper also has its own `Service` trait and doesn't use tower. We can use
-                // `hyper::service::service_fn` to create a hyper `Service` that calls our app through
-                // `tower::Service::call`.
-                let hyper_service = service_fn(move |request: Request<Incoming>| {
-                    // We need to clone here, because oneshot consumes self
-                    tower_service.clone().oneshot(request)
-                });
-
-                if let Err(err) = hyper_util::server::conn::auto::Builder::new(TokioExecutor::new())
-                    .serve_connection_with_upgrades(tls_stream, hyper_service)
-                    .await
-                {
-                    warn!(%err, %remote_addr, "failed to serve connection");
+            let span = tracing::debug_span!("accept tcp connection");
+            tokio::spawn(
+                async move {
+                    let span = tracing::trace_span!(
+                        "accept tls connection",
+                        // otel.name = "accept tls connection",
```

Comment:
nit: Remove that line. Maybe add a comment stating why we don't need that field here.

Reply:
Oh, I thought I had removed it. I think I got stuck wondering what the otel lib does when it is missing. My assumption is that it uses the span name. Either way, looks fine in Grafana/Jaeger.

I think I'll just remove it. We can add an explanation when we do set it perhaps?

Reply:
> Either way, looks fine in Grafana/Jaeger.

Yeah it seems to work fine. I think `otel.name` is only needed when the span name needs to be dynamic.

> I think I'll just remove it. We can add an explanation when we do set it perhaps?

Agreed, that makes more sense.

Reply:
Done in 9cf0316cae86aa34424c49171b2ee80fde01bb38